src/_guides/language/spec.md

```diff
@@ -17,6 +16,20 @@ For a gentler introduction to the Dart language, see the
 You can find previous editions of the specification at
 [Standard ECMA-408](http://www.ecma-international.org/publications/standards/Ecma-408-arch.htm).
 
+
+## Dart 2 changes
+
+Dart 2 is changing the Dart language in many ways,
+some of which are not backward-compatible.
+For information about what is changing and the implementation status,
+see the following resources:
+
+* [Dart 2 Updates](/dart-2.0)
+* [Dart Language and Library Newsletters](https://github.com/dart-lang/sdk/tree/master/docs/newsletter#dart-language-and-library-newsletters)
+* [Language specification source file (TeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
```

Comment:
Maybe put the TeX format last :). Few folks will want to read TeX sources.

Btw, it wouldn't be too difficult to setup a site with a built PDF version of the spec.

Reply:
I actually put the TeX third to bury it. The last item in a list is easier to see, in my experience.

Could you open an (sdk?) issue for publishing a PDF version of the current externally?

Reply:
Here is the issue: https://github.com/dart-lang/sdk/issues/31957

Reply:
If you agree with what is proposed https://github.com/dart-lang/sdk/issues/31957#issuecomment-359309323, then this link could be used for the most recent PDF of the Language Specification: https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/dartLangSpec.pdf

Reply:
@kwalrath - if you prefer I could publish the Language Spec PDF on dartlang.org, and update it periodically. We could then change the link to something like:

* [Language specification](#link-to-PDF-on-dartlang) ([TeX source](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex))

Reply:
Let me ask the people who care about where we should publish the PDF.

Reply:
If do that we should clearly mark it as "draft" as it is WIP. 
Would like the "informal" readme file to give a good overview of current semantics (i.e., what can be found as an informal spec and what has already been added to the draft 5th version of the spec - and hence can be found in the PDF if we publish it.

Reply:
> If we do that, we should clearly mark it as "draft" as it is WIP.

The first page of the PDF has "5th edition draft" right under the title. Is that enough? Otherwise I can probably add "DRAFT" on each page if you prefer.

> Would like the "informal" readme file to give a good overview of current semantics

To be clear, which informal readme are you referring to?

Reply:
Also note that https://www.dartlang.org/dart-2 has a summary of the main Dart 2 updates. Might a link to that page be sufficient?

Reply:
Adding a draft watermark or something would be ideal.

I was referring to https://github.com/dart-lang/sdk/blob/master/docs/language/informal/README.md

Ideally, that file would have an updated table of the status of each file/part the directory to give an easy overview.

Reply:
> Adding a draft watermark or something would be ideal.

Done: https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/dartLangSpec.pdf